src/app/Main/Header/Header.component.tsx

```diff
@@ -63,15 +63,6 @@ const rightHeaderStyle = css`
     display: flex;
     align-items: flex-end;
 `;
-const subheaderStyle = (open: boolean): SerializedStyles => css`
```

Comment:
Reject: This style belongs to this component because it is for aligning the header elements (just one for now). So please leave it here. You could extract the export button specific part to the HeaderExportButtonComponent (not the HeaderExportContainer).

Reply:
Done. Remove Style from Container.